commercial/app/views/debugger/allcreatives.scala.html

```diff
@@ -645,7 +645,9 @@ <h2 class="test-page-head">
                                 omnitureId: '[%OmnitureID%]'
                             });
 
-                            commercialComponent.postLoadEvents['capi@if(component("type") == "single"){Single}'] = function () {
+                            var oldFn = commercialComponent.postLoadEvents['capi@if(component("type") == "single"){Single}'];
```

Comment:
In this context I'm not sure if `oldFn`'s meaning is immediately obvious, at least until you read a little further. What about `originalPostLoadEvents`?